src/quantum/azext_quantum/operations/job.py

```diff
@@ -234,15 +235,19 @@ def submit(cmd, resource_group_name, workspace_name, location, target_id, job_in
             raise RequiredArgumentMissingError("No storage account specified or linked with workspace.")
         storage = ws.properties.storage_account.split('/')[-1]
     job_id = str(uuid.uuid4())
-    container_name = "quantum-job-" + job_id
-    connection_string_dict = show_storage_account_connection_string(cmd, resource_group_name, storage)
-    connection_string = connection_string_dict["connectionString"]
-    container_client = create_container(connection_string, container_name)
     blob_name = "inputData"
 
+    resource_id = "/subscriptions/" + ws_info.subscription + "/resourceGroups/" + ws_info.resource_group + "/providers/Microsoft.Quantum/Workspaces/" + ws_info.name
+    workspace = Workspace(resource_id=resource_id, location=location)
+
+    knack_logger.warning("Getting Azure credential token...")
+    container_uri = workspace.get_container_uri(job_id=job_id)
```

Comment:
Does anything here create an actual container from the containerUri? Previously, it looks like create_container would do this. 
We need to create a container if it doesn't already exists. The service just returns a URI, it doesn't actually create it today., although we are planning to modify it so that it creates it. The cli should handle both cases.

Reply:
It looks like upload_blob will create a container if it doesn't already exist, so I think we are good.